rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketController.java

```diff
@@ -214,76 +262,111 @@
     return result;
   }
 
+  /**
+   * This endpoint retrieves all Workbasket Access Items for a given Workbasket.
+   *
+   * @title Get all Workbasket Access Items
+   * @param workbasketId the id of the requested workbasket.
+   * @return the access items for the requested workbasket.
+   * @throws NotAuthorizedException if the current user is not member of role BUSINESS_ADMIN or
+   *     ADMIN
+   * @throws WorkbasketNotFoundException the requested Workbasket does not exist.
```

Comment:
We started all other exception annotation with "if", should we add this here too?

Reply:
yes